titus-client/src/main/java/com/netflix/titus/runtime/endpoint/v3/grpc/GrpcJobManagementModelConverters.java

```diff
@@ -21,33 +21,13 @@
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
+import java.util.Optional;
 import java.util.concurrent.TimeUnit;
 import java.util.stream.Collectors;
 
 import com.google.common.base.Preconditions;
 import com.google.protobuf.ByteString;
-import com.netflix.titus.api.jobmanager.model.job.BasicContainer;
-import com.netflix.titus.api.jobmanager.model.job.BatchJobTask;
-import com.netflix.titus.api.jobmanager.model.job.CapacityAttributes;
-import com.netflix.titus.api.jobmanager.model.job.Container;
-import com.netflix.titus.api.jobmanager.model.job.ContainerResources;
-import com.netflix.titus.api.jobmanager.model.job.Image;
-import com.netflix.titus.api.jobmanager.model.job.Job;
-import com.netflix.titus.api.jobmanager.model.job.JobDescriptor;
-import com.netflix.titus.api.jobmanager.model.job.JobGroupInfo;
-import com.netflix.titus.api.jobmanager.model.job.JobModel;
-import com.netflix.titus.api.jobmanager.model.job.JobState;
-import com.netflix.titus.api.jobmanager.model.job.JobStatus;
-import com.netflix.titus.api.jobmanager.model.job.LogStorageInfo;
-import com.netflix.titus.api.jobmanager.model.job.NetworkConfiguration;
-import com.netflix.titus.api.jobmanager.model.job.Owner;
-import com.netflix.titus.api.jobmanager.model.job.ServiceJobProcesses;
-import com.netflix.titus.api.jobmanager.model.job.ServiceJobTask;
-import com.netflix.titus.api.jobmanager.model.job.Task;
-import com.netflix.titus.api.jobmanager.model.job.TaskState;
-import com.netflix.titus.api.jobmanager.model.job.TaskStatus;
-import com.netflix.titus.api.jobmanager.model.job.TwoLevelResource;
-import com.netflix.titus.api.jobmanager.model.job.Version;
+import com.netflix.titus.api.jobmanager.model.job.*;
```

Comment:
make sure to import `codequality/Intellij_code_style.xml` into intellij to get same formatting rules